### ISSUE TITLE
2D/3D contrast problem (player_api related)

### ISSUE THREAD

**Issue by rudzik8** [CasimirKaPazi/Voxelgarden#86]
It looks weird for me: currently, we have: flat mobs, flat torches, flat flowers, flat everything BUT we have player in 3D, with skin from MTG (_Sam_)
As I can see, some time ago Voxelgarden had an original character (and his 2D skin is still in the files and can be enabled by commenting most part of player_api/init.lua!), flat and looking just okay, not breaking vg atmosphere.

In README you wrote:
> I personally can't Blender, and I don't care enough to learn. That's why there are only 2D entities.
...but models for mobs from Minetest-c55 (Dungeon Master, Oerkki, Rat, etc) were created long time ago, you can just take them from Mobs Monster and use it without learning Blender (but VG-only mobs like omsk, 1hour will be still flat)
But personally, I love that Voxelgarden keeps 2D stuff from older versions of Minetest Game, and wouldn't like to see more 3D stuff in VG

I think there are 2 real ways to fix 2D/3D contrast:
1. Make character use 2D skin again and tweak physics/autojump for that
2. Draw a new skin for player based on old 2D one, so at least main character won't be Sam from MTG

2nd way will be most easy from my vision, BUT we will still have 3D character among 2D entities, torches and flowers
I don't think you would make every mob use 3rd-party model (that's just too hard and dumb), so that 3rd way isn't listed above

**Comment by rudzik8** [comment]
Some screenshots:

![screenshot_20220416_131138](https://user-images.githubusercontent.com/87843378/163664491-07f78059-6b3d-4a48-86d7-29e70135d630.png)
_Character by default (Sam)_

![screenshot_20220416_131058](https://user-images.githubusercontent.com/87843378/163664490-4b0c1b92-5d11-4263-b162-c3ccd1441556.png)
_Flat character, enabled by tweaking player_api a bit_

![screenshot_20220416_132811](https://user-images.githubusercontent.com/87843378/163664486-a8dd8606-f7d4-481e-bd47-4659e72cd447.png)
_My attempt at making old original character look nicely in 3D (based on Sam, of course)_

**Comment by CasimirKaPazi** [comment]
Yes, that's an issue. But I don't know how to deal with it. I think that for multiplayer it is very useful to have 3D characters, so players can see what others do.

I added a setting to enable the 2D sprite in 10d68ee. (I thought I had done this before but apparently not.)
If you provide that new character design I will add it into the game.
